### ISSUE TITLE
Allow other non-admins to write announcements

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/announcementcenter#5]
Requested by @Ark74 in https://github.com/owncloud/announcementcenter/issues/37

Would be cool if non-admins can also write announcements.

**Comment by nickvergessen** [comment]
Admin group can currently be set via Console only:
```
./occ config:app:set announcementcenter admin_group --value ug-test
```

UI will come when we have the sidebar for system tags

**Comment by nickvergessen** [comment]
Implemented for 2.0 which will ship aligned with the next Nextcloud major release.
